journeys/et/updateIP.ts

```diff
@@ -0,0 +1,25 @@
+import {Journey} from "types/journey"
+import {readFileSync, writeFileSync} from "fs";
+import {sep} from "path";
+import {execCommand} from "app/helpers";
```

Comment:
We should probably look into officially enforcing some styling rules, personally I prefer spaces between the imports (like `import { execCommand } from....`. I usually use "StandardJS" but I'll admit its not setup in my local environment, maybe we should?

Also get rid of semicolons [JS doesn't need them (and if you run into a case where it does, its not good code)](https://standardjs.com/rules.html#semicolons)

Reply:
Good idea to set something like that up. I'll fix it for now.